hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/shortcircuit/TestShortCircuitCache.java

```diff
@@ -957,6 +958,83 @@ public void testDomainSocketClosedByDN() throws Exception {
     }
   }
 
+  // Regression test for HDFS-16473
```

Comment:
I don't quite understand the connection between this UT and HDFS-16473.

Reply:
Oops, I gave a wrong JIRA. Thanks for catching this! It should be HDFS-16535.